### PR TITLE
Feature/sr 9 implement sign in form

### PR DESCRIPTION
**Pull request by mzajaczkowsky**
- Add Sign In page
- Create Route /login for Sign In form

### CODE REVIEW

File: src/pages/Login.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+export function Login() {
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+
+  const manageSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
```
Comment (masterHAWK99) [inline src/pages/Login.tsx:8]: Use meaningful names, not `e`, also return type is redundant.

File: src/pages/Login.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    e.preventDefault();
+    console.log(`${email} | ${password}`);
```
Comment (masterHAWK99) [inline src/pages/Login.tsx:10]: Please get rid of that debug

File: src/pages/Login.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+export function Login() {
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
```
Comment (masterHAWK99) [inline src/pages/Login.tsx:6]: Type of these states are infered from inistial value, so generics are not necessary.

File: src/pages/Register.tsx [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+              />
+            </div>
+            <div className="signup__wrapper2__pass">
+              <div className="text-caption">Password</div>
+              <input
+                className="input-text"
+                type="password"
+                placeholder="password"
+                value={password}
+                onChange={(e) => setPassword(e.target.value)}
```
Comment (masterHAWK99) [inline src/pages/Register.tsx:49]: Same as in login, and shouldn't be state for name yet?

File: src/pages/Signin.tsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import { Login } from "./Login";
+
+export function Signin() {
+  return (
+    <div className="signin">
+      {/* navbar will be placed there */}
+      <Login />
+    </div>
+  );
+}
```
Comment (masterHAWK99) [inline src/pages/Signin.tsx:10]: The same navbar will be on all pages, so it should be implemented using `Outlet`, not here.

File: src/pages/Signup.tsx [hunk 6]
```diff
@@ -0,0 +1,10 @@
+import { Register } from "./Register";
+
+export function Signup() {
+  return (
+    <div className="signup">
+      {/* navbar will be placed there */}
+      <Register />
+    </div>
+  );
```
Comment (masterHAWK99) [inline src/pages/Signup.tsx:9]: Same as Signin

**Review by masterHAWK99 — CHANGES_REQUESTED** [review]

**Review by GalacticOctopus — APPROVED** [review]
Looking good

**Pull request merged**